src/goLanguageServer.ts

```diff
@@ -291,13 +280,11 @@ export function parseLanguageServerConfig(): LanguageServerConfig {
 }
 
 /**
- * Get the absolute path to the language server to be used.
- * If the required tool is not available, then user is prompted to install it.
- * This supports the language servers from both Google and Sourcegraph with the
- * former getting a precedence over the latter
+ * If the user has enabled the language server, return the absolute path to the
+ * correct binary. If the required tool is not available, prompt the user to
+ * install it. Only gopls (Google) and go-langserver (Sourcegraph) are supported.
```

Comment:
nit: time to remove mention of `go-langserver` in this comment.

Reply:
Done.